src/graphql/types/User.ts

```diff
@@ -6,7 +6,7 @@ const User = objectType({
   description: 'A Geektext user.',
   definition(t) {
     t.id('id');
-    t.emailAddress('email_address');
+    t.emailAddress('email');
```

Comment:
This was tested to make sure changing the name doesn't break anything, right?

Reply:
yeah I’m an idiot and email_address wasn’t the name in the database, so this was actually breaking things